### DISCUSSION TITLE
File Preview in Discord Embed

### DISCUSSION THREAD

**Discussion by byPixelTV** [diced/zipline#497] [Questions]
When you upload a file that a preview gets displayed in the Discord embed like this https://link.bypixeltv.xyz/view/T8Nq09.png

**Comment by diced** [comment]
doesn't this happen already..?

**Reply by Felitendo** [reply]
For pictures it does, but not for videos

**Reply by byPixelTV** [reply]
If I enable the embedded option it looks like this https://link.bypixeltv.xyz/view/kGpVZw.png. Do I have to add something to my docker-compose.yml for it to work? This is my docker-compose.yml https://link.bypixeltv.xyz/r/1PL2OL.yml?compress=false?

**Reply by diced** [reply]
If you are on 3.7.6+ try out 3.7.5 and see if the issue still happens, to narrow it down to changes made recently.

**Reply by byPixelTV** [reply]
Sorry if this is a dumb question but how can I change the version of Zipline?

**Reply by AdamK2003** [reply]
you can put the version number (or another tag name) after a `:` in the image directive in docker-compose.yml, for example `image: ghcr.io/diced/zipline:3.7.5`

**Comment by byPixelTV** [comment]
> If you are on 3.7.6+ try out 3.7.5 and see if the issue still happens, to narrow it down to changes made recently.

Now I am on 3.7.5 and it still looks like this https://link.bypixeltv.xyz/view/Jj3yFA.png it seems like only the /view/ urls get embedded but in this weird way

**Reply by diced** [reply]
What shows up when you replace the `view` in those URLs to `r`? Does the image display correctly or is there an error displayed. Since it's happening on both versions, I think you can go back to the latest version and test it out.

**Reply by byPixelTV** [reply]
It still looks like this in version 3.7.7 https://link.bypixeltv.xyz/view/Zy2cOa.png and where the image is I used https://link.bypixeltv.xyz/u/d9Ycv2.png I think

**Comment by migracing** [comment]
Whenever I share on discord i have to replace the `u` with `view` in order for the video preview to show. Is there a way to have the link use `view` every time i copy the URL? I've searched the documentation and haven't seen an option for it.  

Do I just need to try an older version?

I'm new to this so any help is appreciated. Thank you.
